solutions/usaco-736.mdx

```diff
@@ -2,87 +2,87 @@
 id: usaco-736
 source: USACO Bronze 2017 US Open
 title: Bovine Genomics
-author: Mrinall Umasudhan, Ananth Kashyap
+author: Mrinall Umasudhan, Ananth Kashyap, Ben Dodge
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_cownomics_bronze_open17.html)
 
-## Solution 1
+## Solution 1 - Brute force
 
-<LanguageSection>
-
-<JavaSection>
+We'll iterate through every character in every genome. For each character index, we'll find if there are any characters that appear in the genome of both a spotted and plain cow at that index. If there are, this position isn't a possible solution, so we can end this part of our search and move on to the next position to save on runtime. If there aren't, this is a possible solution. 
 
 ## Implementation
+**Time Complexity**: $\mathcal O(MN^2)
```

Comment:
```suggestion
**Time Complexity**: $\mathcal{O}(MN^2)
```

Reply:
I edited the comment to include the colon inside of the bold style, as per https://usaco.guide/general/adding-solution#steps